elements/rh-menu-dropdown/rh-menu-dropdown.css

```diff
@@ -0,0 +1,135 @@
+:host {
+  position: relative;
+  display: inline-block;
+}
+
+#menu-list {
+  position: fixed;
+  padding: 8px 0;
+  margin: 0.25rem 0;
+  display: none;
+  z-index: 1000;
+  min-width: 296px;
+  max-width: 100%;
+  border-radius: 3px;
+  border: 1px solid #c7c7c7;
+  background: #ffffff;
+  box-shadow: 0 4px 6px 1px rgba(21, 21, 21, 0.25);
+}
+
+:host([open]) #menu-list {
+  display: block;
+}
+
+.visible {
+  display: block;
+}
+
+#menu-toggle {
+  color: #151515;
+  font-family: var(--rh-font-family-body-text, RedHatText, 'Red Hat Text', Helvetica, Arial, sans-serif);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  width: 100%;
+  padding: 6px 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  max-height: 36px;
+  border-radius: 3px;
+  background: #ffffff;
+  border: none;
+
+  .action-icon rh-icon {
+    --rh-icon-size: 10px;
+  }
+
+  .info-section {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  &.boxed {
+    border: 1px solid #c7c7c7;
+
+    &:not(.open) {
+      &:hover,
+      &:active {
+        border-color: transparent;
+        box-shadow: 0 0 0 2px #0066cc;
+      }
+
+      &:focus {
+        outline: 2px solid #0066cc;
+        outline-offset: 4px;
```

Comment:
When the toggle's collapsed, the `outline-offset` should be 2px (`--rh-length-3xs`).